.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Run SpatialPy Integration Tests
+
+on:
+  push:
+    branches: [staging]
```

Comment:
Since this is the "from" branch, I suggest we change this to "develop".  Then, any PR that goes to either "staging" or "main" will run this test.